examples/js/loaders/MMDLoader.js

```diff
@@ -4407,17 +4413,78 @@ THREE.MMDHelper.prototype = {
 
 	},
 
-	renderOutline: function ( scene, camera ) {
+	renderOutline: function () {
 
-		var tmpEnabled = this.renderer.shadowMap.enabled;
-		this.renderer.shadowMap.enabled = false;
+		var invisibledObjects = [];
+		var setInvisible;
```

Comment:
Why don't you directly create the  `setVisible` and `restoreVisible` functions here?